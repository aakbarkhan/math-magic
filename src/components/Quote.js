import React from 'react';

function Quote() {
  return (
    <div className="quotes">
      <h1>
        Mathematics is not about numbers, computations,
        or algorithams: it is about understanding. -William Paul Thurston
      </h1>
    </div>
  );
}

export default Quote;
