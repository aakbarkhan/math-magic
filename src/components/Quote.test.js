import React from 'react';
import renderer from 'react-test-renderer';
import Quote from './Quote';

it('renders correctly', () => {
  const tree = renderer
    .create(<Quote page="http://localhost:3000/quotes">Quote</Quote>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
