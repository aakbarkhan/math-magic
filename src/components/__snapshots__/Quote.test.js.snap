// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="quotes"
>
  <h1>
    Mathematics is not about numbers, computations, or algorithams: it is about understanding. -William Paul Thurston
  </h1>
</div>
`;
